src/sentry/tasks/post_process.py

```diff
@@ -388,6 +421,9 @@ def post_process_group(
         group_events: Mapping[int, GroupEvent] = {
             ge.group_id: ge for ge in list(event.build_group_events())
         }
+        if occurrence is not None:
+            for ge in group_events.values():
+                ge.occurrence = occurrence
```

Comment:
Note that there will only ever be one groupevent in this case. Once we port perf issues over, we can get rid of group state and just assume there's only ever one issue